aces/Loris
# [issue_tracker] Introduce Rejected status

## Brief summary of changes

- Add Rejected status to issue tracker. Only someone with `Can re-assign issues, mark issues as closed, comment on all, edit issues.` permission can change the status to this. 

#### Testing instructions (if applicable)

1. When creating an issue, make sure that the rejected status option only appears if you have the permission: 'Can re-assign issues, mark issues as closed, comment on all, edit issues.'
2. Repeat with editing an issue

[CCNA OVERRIDE PR](https://github.com/aces/CCNA/pull/4045)
